src/main/java/io/cdap/plugin/snowflake/common/client/SnowflakeAccessor.java

```diff
@@ -74,6 +75,38 @@ public void runSQL(String query) {
     }
   }
 
+  /**
+   * Returns field descriptors for specified table name
+   * @param schemaName The name of schema containing the table
+   * @param tableName The name of table whose metadata needs to be retrieved
+   * @return list of field descriptors
+   * @throws SQLException If an error occurs while retrieving metadata from the database
+   */
+  public List<SnowflakeFieldDescriptor> getFieldDescriptors(String schemaName, String tableName) throws SQLException {
```

Comment:
can it still throw `SQLException`?

Reply:
No, removed it. 30ba2aabd7b774edc1b13a54868366987ff5b912